### PR TITLE
Requests Console standardization

### LINKED ISSUE #25349: Can't request secondary goals from Medbay in NSS Farragus
### Exploit Reports

- [X] I confirm this issue is not an exploit. (Required)

### BYOND Version

515.1636

### Issue Description

I tried to request secondary goals for Medical from NSS Farragus from both the CMO's request console and the Medical Chemistry request console. The Chemistry console said 'No secondary goals for Chemistry allowed! Secondary goals allowed from Science, Medbay, Botany, Bar, and Chef.' Trying it from the CMO's request console gave the same error message, though it said 'No secondary goal for CMO allowed!' 

### What did you expect to happen?

I expected to be able to order secondary goals from Medbay request terminals.

### What happened instead?

I was told that the terminals I was requesting Secondary Goals from were Chemistry and CMO's Office, not Medbay, and thus was unable to.

### Why is this bad/What are the consequences?

Secondary Goals cannot be obtained by Medical when we're stuck on the NSS Farragus.

### Steps to reproduce the issue.

When we're on the Farragus, request a Secondary Goal from either of those medbay computers.

### When did the problem start happening?

Not sure, since I've never gotten a Secondary Goal on the Farragus. I think the system is new?

### Extra information

My guess would be that the Farragus request terminals are misnamed compared to those on other stations?

### Relevant log output/runtime error

_No response_

### PR DESCRIPTION
**Pull request by FunnyMan3595**
## What Does This PR Do
Requests Consoles have been standardized across all four maps:
Chemistry, Mining, Paramedic, Psychiatrist, and Xenobiology Requests Consoles have been separated out from their general department.
Several wrong-department Requests Consoles were fixed.
A bunch of missing Requests Consoles have been added.
`name` has been unset on all Requests Consoles, as it gets overridden in Initialize anyway.
Secondary Goals was updated to reference the new Chemistry Requests Console.
One random fax machine had its department set to IAA office instead of Internal Affairs Office. Corrected.

Fixed #25349

## Why It's Good For The Game
Standardization good. Diversification of Requests Consoles for Secondary Goals also good.

## Testing
Diffed lines with "department =" across the four map files, sorted. StrongDMM and mapmerge2 are happy.
The change to Secondary Goals compiled.

## Changelog
:cl:
tweak: [All maps] Chemistry, Mining, Paramedic, Psychiatrist, and Xenobiology Requests Consoles have been separated out from their general department.
fix: [All maps] Several wrong-department Requests Consoles were fixed.
add: [All maps] A bunch of missing Requests Consoles have been added.
/:cl:

### CODE REVIEW

File: _maps/map_files/stations/boxstation.dmm [hunk 1]
```diff
@@ -656,7 +656,6 @@
 /obj/machinery/requests_console{
 	department = "Security";
 	departmentType = 5;
-	name = "Security Requests Console";
 	pixel_y = 30
```
Comment (GDNgit) [inline _maps/map_files/stations/boxstation.dmm:659]: these are still so snowflaked. we can subtype these, we have the technology!

Comment (FunnyMan3595) [reply _maps/map_files/stations/boxstation.dmm:659]: WYCI

Comment (GDNgit) [reply _maps/map_files/stations/boxstation.dmm:659]: true!

**Comment by 1080pCat** [comment]
map edit with no pictures of changes >:(

**Review by Burzah — APPROVED** [review]
Looks good pending the resolution of one map merge conflict.

**Pull request merged**